mentat/commands.py

```diff
@@ -241,6 +242,42 @@ def help_message(cls) -> str:
         return "Clear the current conversation's message history"
 
 
+SEARCH_RESULT_BATCH_SIZE = 10
+
+
+class SearchCommand(Command, command_name="search"):
+    async def apply(self, *args: str) -> None:
+        stream = SESSION_STREAM.get()
+        code_context = CODE_CONTEXT.get()
+
+        if len(args) == 0:
+            await stream.send("No search query specified\n", color="yellow")
+            return
+        try:
+            query = " ".join(args)
+            results = await code_context.search(query=query)
+        except UserError as e:
+            await stream.send(str(e), color="red")
+            return
+
+        for i, (feature, score) in enumerate(results):
+            _i = f"{i}: " if i < 10 else f"{i}:"
+            await stream.send(f"{_i} {score:.3f} | {feature.path}")
```

Comment:
You can use :2 to force i to take up the same vertical space:
```
await stream.send(f"{i:2} {score:.3f} | {feature.path}")
```
I think that'd be a bit cleaner.